browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -567,14 +573,14 @@ def get_step_info(self, step: int) -> StepInfo:
             if "screenshot" not in self._steps_info[step].obs:
                 try:
                     self._steps_info[step].obs["screenshot"] = np.array(
-                        Image.open(self.exp_dir / f"screenshot_step_{step}.jpg"), dtype=np.uint8
+                        Image.open(self.exp_dir / f"screenshot_step_{step}.png"), dtype=np.uint8
```

Comment:
Still missing, to avoid keeping files open unnecessarily:
```python
with Image.open(self.exp_dir / f"screenshot_som_step_{step}.png") as fd:
   self._steps_info[step].obs["screenshot"] = np.array(fd, dtype=np.uint8)
```